web-ui/src/main/resources/catalog/js/CatController.js

```diff
@@ -1841,6 +1841,15 @@
                   : "";
             return angular.isFunction(this[fnName]) ? this[fnName]() : false;
           },
+          canViewMetadataHistory: function () {
+            var profile = gnConfig["metadata.history.accesslevel"] || 'Editor',
+              fnName =
+                profile !== ''
+                  ? 'is' + profile[0].toUpperCase() + profile.substring(1) + 'OrMore'
+                  : '';
+            console.log("User profile to view history: "+profile);
```

Comment:
This seems to be for debugging - maybe it should be removed?

Reply:
yes, i was using it for local testing. It can be removed